### PR TITLE
Add documentation creation

### PR DESCRIPTION
**Pull request by ennru**
References #29

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,27 +1,64 @@
...
+    paradoxGroups := Map("Language" -> Seq("Java", "Scala")),
+    ApidocPlugin.autoImport.apidocRootPackage := "akka",
+    resolvers += Resolver.jcenterRepo,
+    publishRsyncArtifacts += (makeSite.value -> "www/"),
+    publishRsyncHost := "akkarepo@gustav.akka.io",
+    apidocRootPackage := "akka")
 
-lazy val root = Project(id = "akka-projection", base = file(".")).aggregate(akkaProjectionCore, akkaProjectionTestkit)
+lazy val root = Project(id = "akka-projection", base = file("."))
+  .aggregate(core, testkit)
```
Comment (ignasi35) [inline build.sbt:59]:
```suggestion
  .aggregate(core, testkit, docs)
```

🤔

File: project/CopyrightHeader.scala [hunk 2]
```diff
@@ -1,9 +1,3 @@
-/*
- * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
- */
-
-package akka.grpc
-
 import sbt._, Keys._
```
Comment (ignasi35) [inline project/CopyrightHeader.scala:1]: Hmm, do we really want to remove the header? Shouldn't we have one dated correctly, though?

File: project/project-info.conf [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        url: "https://gitter.im/akka/akka"
+      }
+    ]
+  }
+  core: ${project-info.shared-info} {
+    title: "Akka Projection"
+    jpms-name: "akka.projection.core"
+    levels: [
+      {
+        readiness: CommunityDriven
```
Comment (ignasi35) [inline project/project-info.conf:40]: Should this be `Incubating` (https://github.com/lightbend/lightbend-technology-intro-doc/blob/master/docs/modules/getting-help/pages/support-terminology.adoc) instead?

File: project/CopyrightHeader.scala (hunk 2 above)
Comment (ennru) [reply project/CopyrightHeader.scala:4]: We aren't too good att headers in build files. But I added it back.

File: project/project-info.conf (hunk 3 above)
Comment (ennru) [reply project/project-info.conf:40]: That's a nuance. Maybe when it makes it to a release ;-)

File: docs/src/main/paradox/overview.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  symbol=AkkaVersion
+  value=$akka.version$
+  group1=com.typesafe.akka
+  artifact1=akka-stream_$scala.binary.version$
+  version1=AkkaVersion
+}
+
+Note that it is important that all `akka-*` dependencies are in the same version, so it is recommended to depend on them explicitly to avoid problems with transient dependencies causing an unlucky mix of versions.
+
+The table below shows Akka Projection’s direct dependencies and the second tab shows all libraries it depends on transitively.
```
Comment (seglo) [inline docs/src/main/paradox/overview.md:26]:
```suggestion
The table below shows Akka Projection's direct dependencies and the second tab shows all libraries it depends on transitively.
```

**Comment by ignasi35** [comment]
Oh, wow! I can commit @seglo's suggestion on @ennru's PR. I never noticed that was possible (and thought only the PR author could merge suggestions).

**Pull request merged**